Cargo.toml

```
@@ -23,6 +23,21 @@ members = [
     "tests/deps-helper",
 ]
 
+[workspace.package]
+version = "0.4.0-alpha.17"
+authors = ["Embark <opensource@embark-studios.com>"]
+edition = "2018"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/EmbarkStudios/rust-gpu"
+
+[workspace.dependencies]
+spirv-std = { path = "./crates/spirv-std", version = "=0.4.0-alpha.17" }
+spirv-std-types = { path = "./crates/spirv-std/shared", version = "=0.4.0-alpha.17" }
+spirv-std-macros = { path = "./crates/spirv-std/macros", version = "=0.4.0-alpha.17" }
+spirv-builder = { path = "./crates/spirv-builder", version = "=0.4.0-alpha.17" }
+rustc_codegen_spirv = { path = "./crates/rustc_codegen_spirv", version = "=0.4.0-alpha.17", default-features = false }
```

I think it's a bit confusing to do `default-features = false` here only for this crate, but not e.g. `spirv-builder` - overall, only doing versioning/paths through this mechanism, and leaving `default-features = false` to the individual `Cargo.toml` files, is clearer (and requires fewest amount of changes from the current state of the PR.

If you want to do it:
```suggestion
rustc_codegen_spirv = { path = "./crates/rustc_codegen_spirv", version = "=0.4.0-alpha.17" }
```

---

That doesn't work. The reason I added `defeault-features=false` in the workspace is because without having it it conflicts with a crate specifying a feature other than the default

---

Wait maybe my wording is not entirely correct, but I did get compile errors when running the tests because of conflicting features in `rustc_codegen_spirv`